pkg/transformerconfig/transformerconfig.go

```diff
@@ -108,3 +108,12 @@ func MakeDefaultTransformerConfig() *TransformerConfig {
 	}
 	return c
 }
+
+// MakeCompleteTransformerConfig returns a complete TransformerConfig object from either files
+// or the default configs
+func MakeCompleteTransformerConfig(ldr loader.Loader, paths []string) (*TransformerConfig, error) {
+	if ldr == nil || paths == nil || len(paths) == 0 {
+		return MakeDefaultTransformerConfig(), nil
+	}
+	return MakeTransformerConfigFromFiles(ldr, paths)
+}
```

Comment:
This is just argument parsing code for the build command.

Would rather see it in build.go, where we don't even have to make the tcfgLoader if there are no config paths specified.  Making a new public func on the transformerconfig package obligates one to test all the argument permutations - which you did a nice job doing :) -  but it doesn't seem worth it to me as a public method.

Reply:
I see what you mean. The reason of adding this function here is to make `RunBuild` clean, but adding this function to build.go can also achieve that.